src/App/components/Authentication/index.js

```diff
@@ -0,0 +1,41 @@
+import React, {PropTypes} from 'react'
+import login from 'package/utils/login'
+import logout from 'package/utils/logout'
+import Heading from 'package/components/Heading'
```

Comment:
As for the security concern, each sensitive resource is wrapped in the `Authentication` component so if you aren't logged in it gives you a log in option; you can't see sensitive info without loggin in.